test/net2cog/utility.py

```diff
@@ -0,0 +1,47 @@
+"""Simple utility functions used in the regridder test notebook."""
```

Comment:
```suggestion
"""Simple utility functions used in the net2cog test notebook."""
```
I'm introducing shared utility directory in #92 but for now...